### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD029.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#32] [General]
# https://blog.3place.cn/3place/fsd/FSD029.html

https://blog.3place.cn/3place/fsd/FSD029.html

**Comment by picklecai** [comment]
群丑竭尽丑态握揽大权，
非法的压制却在合法地开展，
迷误的世界呈现在我们眼前。
有的抢女人，有的抢牛羊、祭台的圣杯、烛台、十字架，
长年累月还以此自夸，
逍遥法外，安然无恙。
法庭挤满了告状的人，
法官赫赫地高踞宝座，
骚乱的群众激昂沸腾，
像怒潮一样越聚越多。
谁有同党做他的后盾，
尽可将他的丑事遮盖，
孤立无援的无辜的人，
却要被法庭宣判：有罪！

黑白颠倒是常态

**Comment by picklecai** [comment]
全来作难，全都受难，
陛下的尊严也难以保证。

射出去的箭，终究会射中自己。

**Comment by picklecai** [comment]
这个乱世多么可怕！
大家都在互相残杀，
对政府命令，充耳不闻。

原来这个叫：人道主义危机

**Comment by picklecai** [comment]
佣兵们变得忍无可忍，
暴跳如雷，要求给他们发饷，
如果全部发给他们，
他们就会全部逃光。

什么是被债务捆绑？这就是。

**Comment by picklecai** [comment]
外邦虽然还有诸王，
都以为不干己事，袖手旁观。

没来“勤王”就是好的了。大概德国人没有干过清君侧的事。

**Comment by picklecai** [comment]
答应援助我们的款项，
就像水管已经断水。

资金链断裂。

**Comment by picklecai** [comment]
到处都有自立门户的新贵，
他们都要独立行事，
只好眼睁睁看着他们；
我们已经出让了许多权利，
再没有什么权利留给我们。

藩镇割据。

**Comment by picklecai** [comment]
如今哪里没有匮乏存在？
所缺者不同，此处是缺少钱财。

明明问题多多，积重难返，魔鬼却诱导你说只是缺钱。

**Comment by picklecai** [comment]
天性是罪恶，精神是恶魔，
这两者如果互相结合，
就生出“怀疑”、畸形的怪胎。

为什么这么说？

**Reply by picklecai** [reply]
解放天性，放养精神。“闲则生非”，人就会开始怀疑。

**Comment by picklecai** [comment]
皇帝的古老的国度，
它只出了两个大族。
他们威严地守护皇位：
就是神职人员和骑士；

牧师和骑士，原来是显赫的群体，都是皇帝的倚仗。

**Comment by picklecai** [comment]
你摸不着的，就当它远隔千里，
把握不住的，就当作完全乌有，
没计算过的，就以为纯属虚构，
你没称过的，就说它重量很轻，
你没铸造的，就认为不能通行。

月亮，时光，星星，云朵，大航海……
哈哈！在相对范围内，都是这样的愚笨呀。

**Comment by picklecai** [comment]
皇帝：如果你说谎，就送你进地狱！
梅非斯特：通往地狱的道路我自己知道。

初读哈哈大笑：你可知他从哪里来？

细读极恐：如果一个人，本来就来自地狱，你以为能约束他的威胁实际上约束不了他，那你对他就没有任何约束能力。这样你们的关系全由对方说了算。那稍微有点脑子的人，都不敢用这样的人，也不敢和他订立任何合作关系。

这就是皇帝和浮士德的区别。梅菲斯特对浮士德有所求，那浮士德就对他有一定的约束力量。

皇帝手上有啥牌？所以他在梅菲斯特眼里就是一头待宰的肥羊。

这个朝廷上，唯有宰相，从文人常规的仁义道德出发，直觉这事不大对劲。

**Comment by picklecai** [comment]
我们不流汗，
你们要挨冻。

千方百计让流汗流血的人还要流泪。最大的PUA

**Comment by picklecai** [comment]
不管我是怎样自夸，
她骂我是衣裳架子。

衣裳架子，居然既可以夸人，又可以骂人。骂人就是脑袋空空。

**Comment by picklecai** [comment]
他逃避太阳，却想靠寒霜取暖。

这的确是白月光和白米粒、朱砂痣和蚊子血的描画。白月光和朱砂痣虽然寒霜，仍然曲意奉承。

**Comment by picklecai** [comment]
我不谩骂负义的人，
我要为他调毒磨刀；

人而不仁，疾之已甚。

**Comment by picklecai** [comment]
到头来我们这个人世，
虽然演了千万出滑稽戏，
依然是个唯一的大呆子。

乌合之众，群体低智。组织行为学的基本认识就是：组织无智。

**Comment by picklecai** [comment]
智慧
人类最大的两个敌人，
恐惧和希望。

恐惧是敌人可以理解，希望居然也是敌人。

**Comment by picklecai** [comment]
可是到了最近几年，
妇女再也不惯于勤俭，
像一个手头不宽的客家，
金钱有限而欲望很大，
做丈夫的就受苦难挨，
到处望去，全是债台。

她们把纺绩所赚的钱，
花在情夫和服装上面；
还跟一班浪荡的朋友，
大吃大喝，越来越讲究。

这使我越发看重黄金：
我成为男性，改名贪吝！

节俭和贪吝，原来是一体的。

**Comment by picklecai** [comment]
给你们金山铜山，
你们的故态总是不会改变。

江山易改本性难移。人还是这个人，所以钱不能解决问题，人才能解决问题。

**Comment by picklecai** [comment]
我岂不曾结交过世人？
学过、教过空洞的学问？
我理智地说出自己的见解，
就更加引起世人高声反对；

原来浮士德的醒悟并非一夕之间。

**Comment by picklecai** [comment]
你把我当作寓言中的老猫，
叫我火中取栗，为你效劳。

时刻保持怀疑，也算某种清醒。
